docs/source/getting-started/install.md

```diff
@@ -174,10 +174,20 @@ Use the project generator helper utility.
 1.  Open a terminal and execute:
 
     ```bash
-    $ npm install -g yo @plone/generator-volto
-    $ yo @plone/volto
+    npm install -g yo @plone/generator-volto
+    yo @plone/volto
     ```
 
+    > 💡 **HINT** In case the `npm install` command throws an error, which would be most likely due to a permissions issue in your home directory, you will need to reclaim ownership of the `.npm` directory.
+
+    You can do so by executing the following command.
+
+
+    ```shell
+    sudo chown -R $(whoami)  ~/.npm
+    ```
```

Comment:
Now that we are in the docs instead of the `README.md`, we can use MyST syntax instead of Markdown hacks, and replace the above with the following. Hopefully I got the nested MyST syntax and indentation for an enumerated list item correct. The Netlify preview should help.

`````
    ````{hint}
    In case the `npm install` command throws an error, which would be most likely due to a permissions issue in your home directory, you will need to reclaim ownership of the `.npm` directory.

    You can do so by executing the following command.

    ```shell
    sudo chown -R $(whoami)  ~/.npm
    ```
    ````
`````

Reply:
Please incorporate this suggestion.